docs/Home.md

```diff
@@ -45,4 +45,10 @@ Keep in mind, the features are based on _our team's_ requirements. In case the c
 
 Please note that the above examples are not definitive as new and existing libraries keep on emerging and evolving.
 
+How do we vote on an issue? 🗳
```

Comment:
Do you think this would be more clear? Feel free to adapt 

```suggestion

Before an issue can be worked on, it must go through our voting process. 
How do we vote on an issue? 🗳
```